SETUP/GRAPH.md

```diff
@@ -0,0 +1,138 @@
+# Graphs
+
+This file contains a description of the way graphs are created in dproofreaders.
+
+In case of problems, please post to the [DP Site Code](http://www.pgdp.net/phpBB3/viewforum.php?f=32)
+forum at the [primary DP site](https://www.pgdp.net/c).
+
+## Overview
+Graphs are drawn on the browser using a set of functions from [svgGraphs.js](../scripts/svgGraphs.js) that interact with the [d3js](https://d3js.org/) library to render [Scalable Vector Graphics (svg)](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics) elements. The 4 types of graphs that are supported are
+1. Bar
+1. Line
```

Comment:
For humans reading this in source code form, not just rendered form, please increment the numbers. Or, since the order doesn't matter, ideally change them to a bulleted list `*`.

Reply:
Ditto below.